README.md

```diff
@@ -12,49 +12,49 @@ Join the [SFXD Discord Community](https://discordapp.com/invite/JG4Bc4q)!
 
 # Community Repos
 
-- [Timeline LWC](https://github.com/deejay-hub/timeline-lwc)
+- [Timeline](https://github.com/deejay-hub/timeline-lwc) (Dave Norris, [@deejay-hub](https://github.com/deejay-hub))
 
   An interactive timeline for the Salesforce platform. https://medium.com/@dave_norris
 
-- [lwc-utils](https://github.com/tsalb/lwc-utils)
+- [LWC Utils and Design Patterns](https://github.com/tsalb/lwc-utils) (James Hou, [@tslab](https://github.com/tsalb))
 
   A set of utilities and patterns for Flexipage SPAs.
 
-- [lightning-chatter-messenger](https://github.com/shunkosa/lightning-chatter-messenger)
+- [Lightning Chatter Messenger](https://github.com/shunkosa/lightning-chatter-messenger) (Shun Kosaka, [@shunkosa](https://github.com/shunkosa))
 
-  Utility item to support real time private chatter conversations.
+  An utility item to support real time private chatter conversations.
 
-- [Salesforce-Custom-Path-Assistant-LWC](https://github.com/maaaaarco/Salesforce-Custom-Path-Assistant-LWC)
+- [Salesforce-Custom-Path-Assistant-LWC](https://github.com/maaaaarco/Salesforce-Custom-Path-Assistant-LWC) (Marco Zeuli, [@maaaaarco](https://github.com/maaaaarco))
 
-- [User Interactions with Platform Events and LWC](https://github.com/lucianostraga/userInteractions-platformEvents-lwc)
+  A custom path assistant built using only Lightning Web Components
 
-- [LWC Listview](https://github.com/shliachtx/lwc-listview)
+- [User Interactions with Platform Events and LWC](https://github.com/lucianostraga/userInteractions-platformEvents-lwc) (Luciano Straga, [@lucianostraga](https://github.com/lucianostraga))
 
-  A dynamic datatable for use in flexipages or standalone 
+  An user interaction tracker with Lightning Web Components and Platform Events for Lightning Experience and Communities
+
+- [LWC Listview](https://github.com/shliachtx/lwc-listview) (Menachem Lazaroff, [@shliachtx](https://github.com/shliachtx))
 
-- [LWC Lookup Datatable](https://github.com/SFXD/LWC-Lookup-Datatable)
```

Comment:
Looks like we lost this in the shuffle, was that intentional?

Reply:
Sorry, recovered!

Reply:
All good!